testing/internal/e2e/tests/static/session_end_delete_project_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package static_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/session"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCliSessionEndWhenProjectIsDeleted tests that an active session is canceled when the respective
+// project for the session is deleted.
+func TestCliSessionEndWhenProjectIsDeleted(t *testing.T) {
+	e2e.MaybeSkipTest(t)
+	c, err := loadConfig()
+	require.NoError(t, err)
+	bc, err := boundary.LoadConfig()
+	require.NoError(t, err)
```

Comment:
Forgive my ignorance, but what is the difference between these two? Maybe we should call the first one `loadTestConfig` in light of the existence of the second one?

Reply:
Renamed for clarification